tools/bumbershoot/idpassemble.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<tool id="idpassemble" name="idpAssemble" version="@VERSION@.1">
+<tool id="idpassemble" name="idpAssemble" version="@VERSION@.2">
```

Comment:
can you revert this, its not needed right? I will upload by hand

Reply:
It is needed to test that the other commit actually has the intended effect. But it didn't work for the PR because PRs don't trigger uploads. :/ I figured a wrapper number bump was pretty harmless, but I guess it could be an extra space newline too.